core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ActiveDirectoryUserStoreConstants.java

```diff
@@ -99,9 +99,6 @@ public class ActiveDirectoryUserStoreConstants {
         setProperty(UserStoreConfigConstants.disabled, "Disabled", "false",
                 UserStoreConfigConstants.disabledDescription,
                 new Property[] { BASIC.getProperty(), BOOLEAN.getProperty(), TRUE.getProperty() });
-        setProperty(UserStoreConfigConstants.CASE_INSENSITIVE_USERNAME, "Case Insensitive Username", "true",
-                UserStoreConfigConstants.CASE_INSENSITIVE_USERNAME_DESCRIPTION,
-                new Property[] { USER.getProperty(), BOOLEAN.getProperty(), TRUE.getProperty() });
```

Comment:
I don't think removing the whole property will work. We just need to avoid showing this in the UI and removing `new Property[] { USER.getProperty(), BOOLEAN.getProperty(), TRUE.getProperty() }` will do that.